A14-channelz.md

```diff
@@ -728,8 +736,39 @@ message ChannelData {
   google.protobuf.Timestamp last_call_started_timestamp = 7;
 }
 
+// A trace event is an interesting thing that happened to a channel or
+// subchannel, such as creation, address resolution, subchannel creation, etc.
+message ChannelTraceEvent {
+  // High level description of the event.
+  string description = 1;
+  // Status/error associated with this event. 
+  google.rpc.Status status = 2;
+  // When this event occurred.
+  google.protobuf.Timestamp event_timestamp = 3;
+  // The connectivity state the channel was in when this event occurred.
+  ChannelConnectivityState state = 4;
+  // uuid of referenced subchannel.
+  // Optional, only present if this event refers to a child object. For example,
+  // this field would be filled if this trace event was for a subchannel being
+  // created.
+  int64 child_channel_id = 5; 
+}
+
+message ChannelTraceData {
+  int64 channel_id = 1;
+  int64 num_events_logged = 2;
```

Comment:
Is this the total number of events logged?   It might be worth clarifying that this may be different than than size of `events` below and why.

Reply:
Done